config/routes.rb

```diff
@@ -71,7 +71,8 @@
   delete '/edit-profile-photo' => 'users#delete_photo'
 
   resource :competitions do
-    get '/shark-tank', action: 'shark_tank', as: 'shark-tank'
+    # hide shark tank for now
```

Comment:
⚠️ Decided to comment out instead of redirect in the controller since it was never publicly exposed in prod -- let me know if you prefer a redirect to the home page instead :)

Reply:
this is fine with me since we're gonna turn this back in in a few days, lol